docs/src/content/docs/guides/make-onetime-payment.mdx

```diff
@@ -121,28 +98,140 @@ Add the following properties:
   chunk='3'
 />
 
-### 7. Request an Outgoing Payment grant
+### 5. Request an interactive Outgoing Payment grant
 
-Request an `outgoingPayment` grant from the sender wallet's authorization server.
+Request an `outgoingPayment` grant from the sender wallet's authorization server. This request requires an interactive grant as the sender will need to consent before an `outgoingPayment` is made against their wallet.
 
 Add the `limits` object with the following properties:
 
 - `debitAmount` : the maximum amount to be deducted from the sender's wallet.
 - `receiveAmount` : the maximum amount to be received in the recipient's wallet.
 
-:::note
-This request requires an interactive grant as the sender will need to consent before an `outgoingPayment` is made against their wallet.
+:::note[Limits]
```

Comment:
The instructions say to add debitAmount and receiveAmount, but then the note says it's optional. Is there a best practice they should follow for a standard one-time payment?